configs/configurations/test_restart_ioc_when_pv_in_alarm/iocs.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
+<iocs xmlns="http://epics.isis.rl.ac.uk/schema/iocs/1.0" xmlns:ioc="http://epics.isis.rl.ac.uk/schema/iocs/1.0" xmlns:xi="http://www.w3.org/2001/XInclude">
+	<ioc name="SIMPLE" autostart="true" restart="false" remotePvPrefix="" simlevel="none">
```

Comment:
Can we set `restart` to true as well as `autostart`? The behaviour when only one of them is set is quite complicated, and ultimately on instruments we pretty much always set either both (very common) or neither (less common).